ydb/_grpc/grpcwrapper/ydb_topic_public_types.py

```diff
@@ -68,6 +72,30 @@ class PublicMeteringMode(IntEnum):
     REQUEST_UNITS = 2
 
 
+class PublicAutoPartitioningStrategy(IntEnum):
```

Comment:
Do you need copy-paste enum with class AutoPartitioningStrategy?
What about inherit internal type from public?